tests/stress/fun_call_limit.py

```diff
@@ -16,14 +16,14 @@ def test(n):
 
 
 # If the port has at least 32-bits then this test should pass.
-print(test(29))
+print(test(28))
 
 # This test should fail on all ports (overflows a small int).
 print(test(70))
 
 # Check that there is a correct transition to the limit of too many args before *args.
 reached_limit = False
-for i in range(30, 70):
+for i in range(29, 70):
```

Comment:
Please apply suggestions from the other PR:
- change the comment above this line to something like "This is the biggest number that will pass on a 32-bit port using object representation B, which has 1<<28 still fitting in a positive small int."
- the for i in range(30, 70): below should be changed to for i in range(28, 70):, so that it tests the transition from non-syntaxerror to syntaxerror (it should've been range(29,70) prior to this PR!)